doc/model/train-se-e3-tebd.md

```diff
@@ -0,0 +1,76 @@
+# Descriptor `"se_e3_tebd"` {{ pytorch_icon }} {{ dpmodel_icon }}
+
+:::{note}
+**Supported backends**: PyTorch {{ pytorch_icon }}, DP {{ dpmodel_icon }}
+:::
+
+The notation of `se_e3_tebd` is short for the Deep Potential Smooth Edition (DeepPot-SE) constructed from all information (both angular and radial) of atomic configurations.
```

Comment:
This definition seems to be se_a?

Reply:
While I modified this from se_e3, it seems wrong as well.

Reply:
Fixed, anyway.

Reply:
se_e3 is from #537. @wanghan-iapcm